kolibri/core/assets/src/views/icon-button/index.vue

```diff
@@ -48,13 +52,27 @@
       type: {
         type: String,
       },
+      alignment: {
+        type: String,
+        default: 'left',
+        required: false,
+        validator(value) {
+          return value === 'left' || value === 'right';
+        },
+      }
```

Comment:
cc @jtamiace @rtibbles @christianmemije we should avoid adding logic that makes use of the concepts of "right" and "left" because it makes adding RTL language support very difficult.

Instead, we can use a boolean prop with something like `iconBeforeText` or, if the constant strings are preferred, `iconLocation` with values `before` or `after`.